internal/cmd/run/playwright.go

```diff
@@ -127,7 +129,7 @@ func runPlaywrightInDocker(p playwright.Project, testco testcomposer.Client, rs
 	log.Info().Msg("Running Playwright in Docker")
 	printTestEnv("docker")
 
-	cd, err := docker.NewPlaywright(p, &testco, &testco, &rs, &rs, createReporters(p.Reporters))
+	cd, err := docker.NewPlaywright(p, region.None, &testco, &testco, &testco, &rs, &rs, createReporters(p.Reporters))
```

Comment:
Why `region.None`?

Reply:
I was using regio region.Region for building URL, but URL now comes in a proper format and I forgot to delete region variable. I fixed this issue.